UI/js-src/lsmb/Reconciliation.js

```diff
@@ -0,0 +1,31 @@
+define("lsmb/Reconciliation",
+       ["dojo/_base/declare",
+        "dojo/topic",
+        "dojo/query",
+        "lsmb/Form",
+        "dijit/_Container",
+        "dojo/NodeList-dom",     // To load extensions in query
+        "dojo/domReady!"],
+       function(declare, Topic, Query, Form, _Container) {
+           return declare("lsmb/Reconciliation", [Form, _Container], {
+               update: function(targetValue, prefix) {
```

Comment:
Why did you decide to *replace* the form with the reconciliation object? Are we sure that we will never have a situation where the reconciliation will be *part* of a form? (i.e. where the widget isn't the only thing contained in the form)